src/main/java/plugins/nate/smp/commands/LiveCommand.java

```diff
@@ -0,0 +1,67 @@
+package plugins.nate.smp.commands;
+
+import net.luckperms.api.LuckPerms;
+import net.luckperms.api.LuckPermsProvider;
+import net.luckperms.api.model.user.User;
+import net.luckperms.api.node.types.InheritanceNode;
+import org.bukkit.Bukkit;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.jetbrains.annotations.NotNull;
+import plugins.nate.smp.utils.Permission;
+
+import static plugins.nate.smp.utils.ChatUtils.*;
+
+public class LiveCommand implements CommandExecutor {
+    @Override
+    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String[] args) {
+        if (!sender.hasPermission(Permission.SMP_LIVE_PERMISSION)) {
+            sendMessage(sender, PREFIX + DENIED_COMMAND);
+            return true;
+        }
+
+        Player targetPlayer;
+        if (args.length == 0) {
+            if (!(sender instanceof Player player)) {
+                sendMessage(sender, PREFIX + "Only players can toggle their live status");
```

Comment:
Should be red for an error.
```suggestion
                sendMessage(sender, PREFIX + "&cOnly players can toggle their live status");
```